test/setup-test-env.ts

```diff
@@ -12,8 +12,6 @@ declare global {
       | "toHaveDescription"
       | "toHaveErrorMessage"
     >
-
-    interface JestAssertion<T = any> extends jest.Matchers<void, T>, OmittedMatchers {}
```

Comment:
why is this being removed?

Reply:
It was causing an error after upgrading to latest Remix V2 and TS versions. I don't remember what was the error specifically, but I think either way we're going to overhaul this file entirely with your upcoming testing efforts.